modin/pandas/test/test_io.py

```diff
@@ -3208,6 +3208,15 @@ def test_to_latex():
     assert modin_df.to_latex() == to_pandas(modin_df).to_latex()
 
 
+@pytest.mark.filterwarnings(default_to_pandas_ignore_string)
+def test_to_xml():
+    # `lxml` is a required dependency for `to_xml`, but optional for Modin.
+    # For some engines we do not install it (like for HDK).
+    pytest.importorskip("lxml")
```

Comment:
Why can't we add it to development dependencies?

Reply:
We can, but it's not strictly necessary. I don’t really want to complicate HDK environment, since it has the same code used for defaulting into pandas.

Reply:
I don't think we will complicate HDK env with a new dep. Maybe we could do this in a separate issue.